gen/com/goide/psi/GoIndexOrSliceExpr.java

```diff
@@ -32,4 +33,9 @@
   @Nullable
   PsiElement getRbrack();
 
+  @Nullable
```

Comment:
Such changes are not directly related to simplify code inspection so it's better to make them in separate PR or at least in a separate commit